src/NetAnalyzers/CSharp/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA2246 | Usage | Info | CSharpAssigningSymbolAndItsMemberInSameStatement, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2246)
```

Comment:
@mavasani, I got a warning to add it to unshipped analyzers. But this doesn't seem correct. It's already shipped. The warning occurred after moving it from Core layer to C# layer.

How to get things automated to move it from Unshipped to Shipped in this layer, and to delete  it from Shipped in the Core layer?

Reply:
@Youssef1313 Can you manually move the existing shipped file entry in core layer to the shipped files in language specific layers?